tripal_chado/tests/src/Kernel/Services/TripalEntityLookupServiceTest.php

```diff
@@ -224,16 +225,17 @@ public function testTripalEntityLookupService() {
     // this uses the fallback entity lookup function getDefaultBundle(),
     // because the term for manufacturer_id is NOT a content type.
     // For the fallback lookup we need to also pass the base table.
+    // Note that we don't depend on the order that contacts are published in.
```

Comment:
Why has this PR changes this? Could we not depend on order before and we just got lucky?

Reply:
I am pretty sure it was just that we got lucky 🙈

Reply:
This is an interesting question perhaps - should publish order be predictable?
There is so much going on inside publish and in chado storage that I think it would be fair to never assume an order when testing if we publish more than one entity at a time. 
This is perhaps related to issue #1910 where I want to be able to publish in smaller batches. That may turn out to be one of two things: 1. easy; 2. insanely difficult - I am voting for number 2 😫